app/models/review.rb

```diff
@@ -3,5 +3,8 @@ class Review < ActiveRecord::Base
   belongs_to :user
 
   validates :rating, numericality: {only_integer: true},
-  inclusion: {in: 1..5, message: "Rating must be between 1 - 5"}
+  inclusion: {
```

Comment:
Address all hound violations. If you update the .yml to address them, make a reply on the hound comments